docs/reference/reana-yaml/index.md

```diff
@@ -25,27 +25,27 @@ This reference guide describes how the Four Questions are transcribed into
 ## Understanding `reana.yaml`
 
 The `reana.yaml` file describes the analysis as a computational workflow with
-its inputs, workflow specification, and its outputs.  The overall structure of
+its inputs, workflow specification, and its outputs. The overall structure of
 `reana.yaml` looks as follows:
 
 | Property | Type         | Mandatory?  | Description                                                                                                              |
-| -------- | ----         | ----------  | -----------                                                                                                              |
-| version  | *string*     | *optional*  | Specifies REANA version to which the analysis was written for. For example, "0.6.0".                                     |
-| inputs   | *dictionary* | *optional*  | Specifies all the high-level inputs to the workflow. Can be composed of "files", "directories", "parameters", "options". |
-| workflow | *dictionary* | *mandatory* | Defines computational workflow, using CWL, Serial or Yadage specifications.                                              |
-| outputs  | *dictionary* | *optional*  | Specifies all the high-level outputs of the workflow.  Can be composed of "files" and "directories".                                       |
+| -------- | ------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
Just curious, did you use a tool to autogenerate the table? 😃
Maybe it's something we can add to our requirements.

Reply:
I edited this table manually (with the Preview from Visual Studio Code on the side to see how it looked) 😅

Reply:
Ah ok, it looked so perfect I thought it was autogenerated somehow 😅 
Also the all the `* -> _` replacements were manual?

Reply:
Those were the automatic formatting on Visual Studio Code. I'm not entirely sure what it's using for .md files 🤔